src/config/localeConfig.js

```diff
@@ -1,22 +1,20 @@
 import * as CONFIG from '@/data/keys/configKeys';
 import * as STRINGS from '@/data/keys/stringsKeys';
+import * as countryCodes from '@/data/keys/countryCodes';
```

Comment:
We gebruiken in de JS-SDK ook een lijst met countryCodes. Kunnen we deze niet hergebruiken?

Reply:
Oh jee. Dit wil ik wel, maar dan moet ik nu eerst ff alle country codes bij de naam gaan zoeken.....

Reply:
Heb [dit hippe tooltje](https://www.tracemyip.org/tools/country-code-list-alpha-2-alhpa-3-converter/) gebruikt als hulpmiddel en alles vervangen :)